packages/fxa-content-server/app/scripts/models/account.js

```diff
@@ -372,6 +382,127 @@ const Account = Backbone.Model.extend(
       });
     },
 
+    _fetchAccountProfileResult: null,
+
+    /**
+     * Fetches the account profile from the GET /account/profile on the auth server and caches the
+     * result locally in _accountProfileFetchPromise. This also updates the account model
+     * with the new properties (minus any verificationMethod, verificationReason values)
+     *
+     * @returns {Promise}
+     */
+    async fetchAccountProfile() {
```

Comment:
I'm confused about the need for this function. It seems like the existing `fetchProfile` method should pull back the ecosystem_anon_id value along with other profile data from the profile server. Could we just set the value there?

Reply:
Ah I didn't noticed that this value was returned there. I can remove this bit then.